sorts/bucket_sort.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
Why remove the shebang line?  https://en.wikipedia.org/wiki/Shebang_(Unix)

Reply:
My bad.